pages/pipelines/build_timeouts.md

```diff
@@ -31,8 +31,11 @@ Note that command step timeouts don't apply to trigger steps and block steps.
 
 ## Scheduled job expiry
 
-Scheduled job expiry helps you avoid having lingering jobs that are never assigned to an agent or run. This expiry time is calculated from when a job is created, not scheduled.
+Scheduled job expiry helps you avoid having lingering jobs that are never assigned to an agent or run. This expiry time is calculated from when a job is created, not scheduled. 
```

Comment:
```suggestion
Scheduled job expiry helps you avoid having lingering jobs that are never assigned to an agent or run. This expiry time is calculated from when a job is created, not scheduled.
```
Just dropping a trailing space after the period here, likely left from my initial change.